### PR TITLE
Update minikube documentation

### PR DESCRIPTION
**Pull request by cwansart**
This pull request updates the outdated Minikube documentation using the deprecated `--vm-driver` parameter, which was replaced by `--driver`. It also updates the driver lists in all localizations and removing xhyve which does not exist anymore.

### CODE REVIEW

**Comment by sftim** [comment]
Hi @cwansart and thanks for this PR.
To make reviewing easier, we'd like you to [limit reviews to a single language](https://kubernetes.io/docs/contribute/intermediate/#localize-content).

This is partly because we probably don't have reviewers who speak all of the languages that you want to change, and mostly because we need a signoff from each language team that a PR touches before we can merge it. (The robot that decides on approval can't tell a code-only change from a change that affects the localized text).

Also, different localization teams use different branching strategies - see  https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use

Would you be willing to save the non-English changes for the future and revise this to just change the `en` localization?

**Comment by cwansart** [comment]
Hey @sftim sure, I can do that.

**Comment by cwansart** [comment]
Done. The others will follow. I first need to check for their way of handling pull requests.

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -187,24 +187,26 @@ example, to run version {{< param "fullversion" >}}, you would run the following
...
+You can change the VM driver by adding the `--driver=<enter_driver_name>` flag to `minikube start`.
 For example the command would be.
 ```shell
-minikube start --vm-driver=<driver_name>
+minikube start --driver=<driver_name>
 ```
  Minikube supports the following drivers:
  {{< note >}}
- See [DRIVERS](https://git.k8s.io/minikube/docs/drivers.md) for details on supported drivers and how to install
+ See [DRIVERS](https://minikube.sigs.k8s.io/docs/reference/drivers/) for details on supported drivers and how to install
```
Comment (sftim) [inline content/en/docs/setup/learning-environment/minikube.md]:
```suggestion
See [Drivers](https://minikube.sigs.k8s.io/docs/reference/drivers/) for details on supported drivers and how to install
```

**Comment by spurin** [comment]
Non-member feedback:

/lgtm

The minikube start --driver approach worked as per the advice, in my case I tested with docker

```
james@ubuntu-throwaway-2:~$ minikube start --driver=docker
* minikube v1.8.2 on Ubuntu 18.04
* Using the docker driver based on user configuration
* Downloading preloaded images tarball for k8s v1.17.3 ...
    > preloaded-images-k8s-v1-v1.17.3-docker-overlay2.tar.lz4: 499.26 MiB / 499
* Creating Kubernetes in docker container with (CPUs=2) (2 available), Memory=1993MB (1993MB available) ...
* Preparing Kubernetes v1.17.3 on Docker 19.03.2 ...
  - kubeadm.pod-network-cidr=10.244.0.0/16
* Launching Kubernetes ...
* Enabling addons: default-storageclass, storage-provisioner
* Waiting for cluster to come online ...
* Done! kubectl is now configured to use "minikube"
```

All of the updated links, resolve accordingly and take you to the expected target references. 

On a mac, I can confirm that the step for checking VMX, worked as expected

```
$ sysctl -a | grep -E --color 'machdep.cpu.features|VMX'
machdep.cpu.features: FPU VME DE PSE TSC MSR PAE MCE CX8 APIC SEP MTRR PGE MCA CMOV PAT PSE36 CLFSH DS ACPI MMX FXSR SSE SSE2 SS HTT TM PBE SSE3 PCLMULQDQ DTES64 MON DSCPL VMX EST TM2 SSSE3 FMA CX16 TPR PDCM SSE4.1 SSE4.2 x2APIC MOVBE POPCNT AES PCID XSAVE OSXSAVE SEGLIM64 TSCTMR AVX1.0 RDRAND F16C
```

The guidance for using --driver=none also worked as expected -

```
james@ubuntu-throwaway-2:~$ sudo minikube start --driver=none
* minikube v1.8.2 on Ubuntu 18.04
* Using the none driver based on user configuration
* Running on localhost (CPUs=2, Memory=1993MB, Disk=63340MB) ...
* OS release is Ubuntu 18.04.2 LTS
* Preparing Kubernetes v1.17.3 on Docker 19.03.4 ...
  - kubelet.resolv-conf=/run/systemd/resolve/resolv.conf
    > kubectl.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubeadm.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubelet.sha256: 65 B / 65 B [--------------------------] 100.00% ? p/s 0s
    > kubeadm: 37.52 MiB / 37.52 MiB [---------------] 100.00% 18.59 MiB p/s 2s
    > kubectl: 41.48 MiB / 41.48 MiB [---------------] 100.00% 16.30 MiB p/s 3s
    > kubelet: 106.42 MiB / 106.42 MiB [-------------] 100.00% 26.08 MiB p/s 4s
* Launching Kubernetes ...
* Enabling addons: default-storageclass, storage-provisioner
* Configuring local host environment ...
*
! The 'none' driver provides limited isolation and may reduce system security and reliability.
! For more information, see:
  - https://minikube.sigs.k8s.io/docs/reference/drivers/none/
*
! kubectl and minikube configuration will be stored in /home/james
! To use kubectl or minikube commands as your own user, you may need to relocate them. For example, to overwrite your own settings, run:
*
  - sudo mv /home/james/.kube /home/james/.minikube $HOME
  - sudo chown -R $USER $HOME/.kube $HOME/.minikube
*
* This can also be done automatically by setting the env var CHANGE_MINIKUBE_NONE_USER=true
* Waiting for cluster to come online ...
* Done! kubectl is now configured to use "minikube"
```

Hope this helps the review process.

File: content/en/docs/setup/learning-environment/minikube.md (hunk 1 above)
Comment (spurin) [reply content/en/docs/setup/learning-environment/minikube.md]: @sftim please see my recent comment, hopefully this assists on the technical review aspect.

**Comment by spurin** [comment]
> /approve
> Needs a technical LGTM (I haven't checked this is accurate)

@sftim please see my recent comment, hopefully this assists on the technical review aspect.

**Comment by sftim** [comment]
/hold
pending v1.18 Kubernetes release

**Pull request merged**